### ISSUE TITLE
How to get h36m_coco_test.json and h36m_coco_train.json?

### ISSUE THREAD

**Issue by qingshi9974** [open-mmlab/mmpose#1419]
We want to train a topdown model for h36m dataset.  However, h36m_coco_test.json and h36m_coco_train.json in the /configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/h36m/hrnet_w32_h36m_256x256.py are missing.

**Comment by ly015** [comment]
Please use this script https://github.com/open-mmlab/mmpose/blob/master/tools/dataset/h36m_to_coco.py

**Comment by qingshi9974** [comment]
> Please use this script https://github.com/open-mmlab/mmpose/blob/master/tools/dataset/h36m_to_coco.py

I want to know how to divide the training set and test set of H36M in Topdown training? Is it the same as in 3D? For example, "S1,5,6,7,8" is the training set, and "S9,S11' is the test set.

**Comment by ly015** [comment]
Yes, that is how we split the dataset.
